pcdsdevices/signal.py

```diff
@@ -422,6 +422,18 @@ def destroy(self):
         return super().destroy()
 
 
+class SummarySignal(AggregateSignal):
+    """
+    Signal that holds a hash of the values of the constituent signals
+
+    Meant to allow tracking of constituent signals via callbacks.
+    The calculated readback value is useless.
+    """
+    def _calc_readback(self):
+        values = tuple(sig.get() for sig in self._signals)
+        return hash(values)
```

Comment:
Could we just return the tuple itself? Or is there good reason not to?

Reply:
I wanted to return an ophyd-compatible data type, due to some issue I had that I can't find any notes on.... 

I suppose there's no real reason to not just return the tuple.  I didn't want people accessing this signal and hoping to get any information from it, since I wasn't grouping any metadata with it (what the values correspond to)